src/com/spikes2212/genericsubsystems/utils/limitationFunctions/Limitless.java

```diff
@@ -5,7 +5,7 @@
 import com.spikes2212.genericsubsystems.BasicSubsystem;
 
 /**
- * This is a {@link Function} that always returns true. This is
+ * This is a {@link Function} from Double to Bollean that always returns true. This is
  * for the constructor of a {@link BasicSubsystem} with no limits.
```

Comment:
u can write something like "an instance of this class can used as the canMove condition in the constructor..."